hw/ip/i2c/doc/_index.md

```diff
@@ -35,6 +34,7 @@ See that document for integration overview within the broader top level system.
 - SCL and SDA ports mapped to I/O via the pinmux.
 - Interrupts in the host mode for FMT and RX FIFO overflow, target NACK, SCL/SDA signal interference, timeout, unstable SDA signal levels, and transaction complete
 - Interrupts in the target mode for TX FIFO empty during a read, TX FIFO nonempty at the end of a read, TX and ACQ FIFO overflow, host sending STOP after ACK, and host ceasing to send SCL pulses during an ongoing transaction
+- SW may reset I2C block using the Reset Manager.
```

Comment:
Technically, this probably is a chip-level thing, not an IP-level thing. But from my POV, it can stay.

Reply:
I can't come up with a way that doesn't seem deliberately obscure to say "the i2c may be reset independently from the top-level, but it's beyond the scope of this document for me to tell you what it's connected to or how to do it."

I'm not particularly familiar with how we document reset logic and distribution in the top-level